model/shared/src/main/scala/explore/model/RootModel.scala

```diff
@@ -18,16 +20,17 @@ import monocle.Lens
 import monocle.macros.Lenses
 import monocle.std.option
 
-import scala.collection.immutable.HashSet
+import scala.collection.immutable.{ HashSet, SortedSet }
 
 @Lenses
 case class RootModel(
-  vault:                 Option[UserVault],
-  tabs:                  EnumZipper[AppTab],
-  focused:               Option[Focused] = none,
-  targetViewExpandedIds: TargetViewExpandedIds = TargetViewExpandedIds(),
-  searchingTarget:       Set[Target.Id] = HashSet.empty,
-  userSelectionMessage:  Option[NonEmptyString] = none
+  vault:                        Option[UserVault],
+  tabs:                         EnumZipper[AppTab],
+  focused:                      Option[Focused] = none,
+  targetViewExpandedIds:        TargetViewExpandedIds = TargetViewExpandedIds(),
+  searchingTarget:              Set[Target.Id] = HashSet.empty,
+  constraintSetViewExpandedIds: SortedSet[ConstraintSet.Id] = SortedSet.empty,
```

Comment:
What do you think of putting all the expanded ids in one class?

Reply:
I guess there's no reason not, too. And just call it expandedIds. I can do that.

Reply:
:+1: